### DISCUSSION TITLE
Create New Branch

### DISCUSSION THREAD

**Discussion by mark-mybaggage** [community/community#88441] [Mobile]
The ability to create a new branch in the Mobile App would be very useful.

Although I use the App a lot, I frequently need to create a branch to trigger a CI/CD pipeline; this causes me to switch to the mobile site - it would be nice to be able to stay in the App.

<details>
GitHub for iOS, Version 1.145.0 (120653769)
Device: iPhone 13 Pro (iOS 17.2.1)
</details>

**Comment by mishmanners** [comment]
Thanks for your suggestion. I have tagged this as Product Feedback so the GitHub Mobile team can check it out.

**Reply by TheArthurAbbott** [reply]
I second this. I was looking everywhere for this feature and getting gaslit by the search engine AI telling me to click things that don't exist. I eventually did the AI's job and searched, found this and feel saner as a result.

This should be a feature accessible from multiple locations. Consider these:

1. Main repo plus symbol drop down
2. Code screen in the triple dot menu
3. Issue screen in development section
4. Pull request plus dropdown
5. Main repo, convert branch name to drop down
6. Change branch screen

There may be better locations, but those are where I thought to look or were erroneously told to look by a gaslighting AI.
